source/chapters/user_interface.rst

```diff
@@ -113,8 +113,9 @@ visually by aligning the beats that appear in each waveform.
    Mixxx default skin (Latenight) - Parallel waveforms
 
 Depending on the skin, Mixxx displays either parallel waveforms (default) or
-separate waveforms. Select your preferred appearance in
-:menuselection:`Preferences --> Interface --> Skin`.
+separate waveforms. You can select your preferred appearance by clicking the 
+gear icon in the upper right corner. Then under Skin Settings, click/toggle 
+the Parallel Waveforms option to choose between parallel or separate waveform.
```

Comment:
```suggestion
separate waveforms. LateNight and Tango skins only have resizable paralel waveforms, Shade skin provides only separate waveforms. In Deere skin you can select your preferred appearance by clicking the 
gear icon in the upper right corner, then toggling the :menuselection:`Skin Settings --> Parallel Waveforms` option.

```

Reply:
Alright. Let me implement this. Thank you :).